fluent-plugin-sql.gemspec

```diff
@@ -17,8 +17,8 @@ Gem::Specification.new do |gem|
   gem.license = "Apache-2.0"
 
   gem.add_dependency "fluentd", [">= 0.12.17", "< 2"]
-  gem.add_dependency 'activerecord', "~> 5.1"
-  gem.add_dependency 'activerecord-import', "~> 0.7"
+  gem.add_dependency 'activerecord', "~> 6.1"
```

Comment:
Memo: It drops older fluentd due to tzinfo version conflict issue, but I think it's O.K.